auth0_flutter/test/web/credentials_extension_test.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<!-- credentials_extension_test.html -->
+<html>
+  <head>
+    <title>credentials_extension_test</title>
+    <link rel="x-dart-test" href="credentials_extension_test.dart">
+    <script src="packages/test/dart.js"></script>
+  </head>
+  <body>
+  </body>
+</html>
```

Comment:
See https://github.com/dart-lang/test/tree/master/pkgs/test#running-tests-with-custom-html

Reply:
This is necessary because the tests use the `WebCredentials` class, which is annotated with `JS()`.